website/docs/api/sentencerecognizer.md

```diff
@@ -12,6 +12,16 @@ api_trainable: true
 A trainable pipeline component for sentence segmentation. For a simpler,
 rule-based strategy, see the [`Sentencizer`](/api/sentencizer).
 
+## Assigned Attributes {#assigned-attributes}
+
+Predicted values will be assigned to `Token.is_sent_start`. The resulting
+sentences can be accessed using `Doc.sents`.
+
+| Location              | Value                                                                                                                                         |
+| --------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Token.is_sent_start` | A boolean value indicating whether the token starts a sentence. After the parser runs this will be `True` or `False` for all tokens. ~~bool~~ |
```

Comment:
```suggestion
| `Token.is_sent_start` | A boolean value indicating whether the token starts a sentence. This will be either `True` or `False` for all tokens. ~~bool~~ |
```